app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -78,7 +78,9 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
     },
 
     getLocation: function () {
-      return this.PIIRemover.stripPIIWithOverride(this.stripGaParam(document.location.href), true, true)
+      // We don't want to remove dates on search pages.
+      var stripDates = !this.getSearchTerm()
```

Comment:
This line gets repeated later - would it be possible to store this value on initialisation?

Reply:
Thanks @andysellick - good point, i've changed it so that it's only called once. Are you able to re-review?